docs/source/api/v2/riak_bucket_bucket_key_key_values.rst

```diff
@@ -0,0 +1,84 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-riak-bucket-bucket-key-key-values:
+
+*********************************************
+``riak/bucket/{{bucket}}/key/{{key}}/values``
```

Comment:
In API 2.x+ this route was renamed to use `vault` instead of `riak` - so the route herein described does not exist.

Reply:
This route should have been removed but that got lost in the rebase. Removed again in 3b76f58348.